inputremapper/injection/macros/macro.py

```diff
@@ -492,11 +490,14 @@ def add_mouse(self, direction, speed):
             "right": (REL_X, 1),
         }[direction.lower()]
 
+        # how long to pause in ms between the injection of mouse events
+        injection_throttle = 10
+
         async def task(handler):
             resolved_speed = value * _resolve(speed, [int])
             while self.is_holding():
                 handler(EV_REL, code, resolved_speed)
-                await self._keycode_pause()
+                await asyncio.sleep(injection_throttle / 1000)
```

Comment:
I will use `await asyncio.sleep(1 / self.mapping.rate)` here, similar to `add_wheel`
That way the event repetition rate is can be controlled by the mapping and the `Mapping.rate` is meant to do that for all `EV_REL` events

Reply:
ok